lib/carrierwave/activejob.rb

```diff
@@ -0,0 +1 @@
+require "carrierwave_backgrounder"
```

Comment:
Might be obvious, but let's add a comment here indicating that we do this to avoid having to rename internals.